Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp

```diff
@@ -284,12 +288,23 @@ void NetworkResourceLoader::startRequest(const ResourceRequest& newRequest)
 }
 
 #if ENABLE(CONTENT_FILTERING)
+static void setSharedParentalControlsURLFilterIfNecessary()
+{
+#if HAVE(BROWSERENGINEKIT_WEBCONTENTFILTER)
+    static std::once_flag onceFlag;
+    std::call_once(onceFlag, [] {
```

Comment:
Can this get called on multiple threads? I doubt it. If not, a static bool would suffice, we don't need the locks inside `call_once`.